setup/depends.sh

```diff
@@ -17,69 +17,64 @@
 # #########################
 #     value
 # #########################
-pgram=brew
+current_path="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-if [ "$(id -u)" != "0" ]; then
-    echo "[!] This script must be run as root." 1>&2
-    exit 1
+if [[ "$EUID" != "0" ]]; then
+  echo "[!] This script must be run as root." 1>&2
+  exit 1
 fi
 
 # -y flag will be passed to this variable for a non-interactive setup.
 SKIP=""
 
-while getopts "y" OPTION
-do
-    case $OPTION in
-        y)
-	    SKIP=" -y"
-            ;;
-    esac
+while getopts "y" OPTION; do
+  case $OPTION in
+    y)
+      SKIP=" -y"
+      ;;
+  esac
 done
 
-if [ "$SKIP" == " -y" ]
-then
-    echo "[*] Non-interactive setup - Setting mysql password to 'your_password'"
-    sudo debconf-set-selections <<< 'mysql-server mysql-server/root_password password your_password'
-    sudo debconf-set-selections <<< 'mysql-server mysql-server/root_password_again password your_password'
+if [[ "$SKIP" == " -y" ]]; then
+  echo "[*] Non-interactive setup - Setting mysql password to 'your_password'"
+  sudo debconf-set-selections <<< 'mysql-server mysql-server/root_password password your_password'
```

Comment:
I have no idea why it's checking for root UID and then prompting for sudo. I left it in just in case, but if the EUID is 0, it means we have root privileges and these sudos should be removed.